### DISCUSSION TITLE
Do i need github pro to buy copilet ?

### DISCUSSION THREAD

**Discussion by chapimenge3** [community/community#109949] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I have just canceled my GitHub pro subscription and my GitHub copilot subscription was canceled along side even though the copilot subscription is valid until August.

Does this mean I have to spend another 100 USD to buy Copilot just because I canceled Github Pro?

**Comment by Bashmdz** [comment]
Yes, you need github pro, please see: https://github.com/orgs/community/discussions/64044

**Comment by davecheney** [comment, marked as answer]
@chapimenge3 no, you do not need a GitHub Pro subscription to purchase Copilot for Individuals.

**Reply by chapimenge3** [reply]
Thank you very much. My GitHub copilet got cancelled when I change my GitHub plan that is why.

**Reply by davecheney** [reply]
@chapimenge3 it looks like there was a problem charging your card. We cannot resolve billing issues via this forum, please contact support https://support.github.com/contact

**Reply by chapimenge3** [reply]
I know there was an issue but until I changed my billing details why would GitHub cancel my copilet subscription as I paid for one year I am entitled to use one year (till August). 

I was gonna update my card details next month.

**Reply by chapimenge3** [reply]
Let's say my card don't work at all does it mean I am should have valid card details for next year today? I mean the next payment is within like 5 month. 

GitHub needs billing details but there is no bill related to GitHub copilet. 


I already contacted support actually.

**Reply by davecheney** [reply]
Thank you. I'm sorry for the delay. We cannot assist you for billing issues via this forum.

**Reply by chapimenge3** [reply]
Thank you for the information @davecheney 🙏
